### DISCUSSION TITLE
paddleocr extraction miss match issue

### DISCUSSION THREAD

**Discussion by tzktz** [PaddlePaddle/PaddleOCR#12380] [Q&A]
@sdcb @ZeyuChen @haobibo @bingooo @llxxxll @tink2123 
code i used..
```
import os
import cv2
from paddleocr import PPStructure,save_structure_res
from paddleocr.ppstructure.recovery.recovery_to_doc import sorted_layout_boxes, convert_info_docx

table_engine = PPStructure(recovery=True, lang='en')

save_folder = './output'
img_path = 'cropped_table_2.png'
img = cv2.imread(img_path)
result = table_engine(img)
save_structure_res(result, save_folder, os.path.basename(img_path).split('.')[0])

for line in result:
    line.pop('img')
    print(line)

h, w, _ = img.shape
res = sorted_layout_boxes(result, w)
convert_info_docx(img, res, save_folder, os.path.basename(img_path).split('.')[0])
```
input image ..
![cropped_table_2](https://github.com/PaddlePaddle/PaddleOCR/assets/99823984/3072ae99-84d4-445c-88d9-31de9bd20a04)

output doc for ocr extraction..
![Screenshot (61)](https://github.com/PaddlePaddle/PaddleOCR/assets/99823984/f413e609-7b53-4a75-b1e8-5dd8a65ca01e)

 
see the miss match in row category details..can anyone resolve it..

**Comment by tink2123** [comment]
Sorry, the effect of the table recognition model is not good enough. We are optimizing and iterating, so you can continue to pay attention.

**Comment by tzktz** [comment]
> Sorry, the effect of the table recognition model is not good enough. We are optimizing and iterating, so you can continue to pay attention.

thank u for ur reply..And consider this issue also..
https://github.com/NielsRogge/Transformers-Tutorials/issues/386
regarding paddle ocr list index out of range..!!
